apps/web/src/features/coaches/components/coach-details/coach-general-info.tsx

```
@@ -1,147 +1,147 @@
+						<div className="flex items-center gap-2 text-muted-foreground text-sm">
+							<Users className="h-4 w-4" />
+							<span>Roles</span>
+						</div>
+						<div className="flex flex-wrap gap-1">
+							{!rolesList && (
+								<span className="text-muted-foreground text-sm">No roles</span>
+							)}
+							{rolesList &&
+								rolesList.map((role: string, index: number) => (
+									<span
+										key={index}
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)